### ISSUE TITLE
Example in allnomagic does not work

### ISSUE THREAD

**Issue by containscafeine** [kedgeproject/kedge#203]
```console
$ kedge generate -f examples/allnomagic/
unable to unmarshal data: could not unmarshal into internal struct: error unmarshaling JSON: json: cannot unmarshal object into Go struct field App.configMaps of type []spec.ConfigMapMod
```

**Comment by surajnarwade** [comment]
@containscafeine , this https://github.com/kedgeproject/kedge/pull/205 will solve this issue

**Comment by containscafeine** [comment]
Closed by #205
